packages/react/leaderboard/src/Leaderboard.tsx

```diff
@@ -374,6 +374,7 @@ export function Leaderboard(props: LeaderboardProps) {
           height={styles?.canvas?.height || defaultChartHeight}
           role="img"
           style={loadingStyles}
+          data-testid="chart-canvas"
```

Comment:
Why is this necessary? Can we not use some other selector than `data-testid`?

Reply:
I was thinking about using `role` since it's the only property we can access I was just a little bit concerned that in a future we add another element with `role="img"`.

But looking at the [testing library suggestions about this](https://testing-library.com/docs/queries/about/#priority) I think we'll be fine, since they recommend using this as first option and if it's necessary you can pass it a `name` as second argument `getByRole('button', {name: /submit/i})`. Let me change it.